src/xmlsig/utils.py

```diff
@@ -101,6 +101,7 @@ def get_rdns_name(rdns):
     data = []
     XMLSIG_NAMEOID_TO_NAME = _NAMEOID_TO_NAME.copy()
     XMLSIG_NAMEOID_TO_NAME[oid.NameOID.SERIAL_NUMBER] = "SERIALNUMBER"
+    XMLSIG_NAMEOID_TO_NAME[oid.NameOID.EMAIL_ADDRESS] = "E"
```

Comment:
Removed from last version of cryptography
https://github.com/pyca/cryptography/commit/0ac3e5037d36ce9409552d57d4a93231ac5d0d4f